Mage/src/main/java/mage/abilities/common/PassAbility.java

```diff
@@ -30,7 +31,7 @@ public PassAbility copy() {
 
     @Override
     public ActivationStatus canActivate(UUID playerId, Game game) {
-        return ActivationStatus.getTrue(this, game);
+        return new ActivationStatus(new ApprovingObject(this, game));
```

Comment:
can you explain this change?

Reply:
I'm not sure actually. I'll check.

Reply:
Alright after check analysis:
The reworked ActivationStatus is now always expecting to have an ApprovingObject when approved, for anything that might want to get additional info.
Having that ApprovingObject referencing the PassAbility does not alter anything, as PassAbilities are do not make use of complex activation logic.
![image](https://github.com/magefree/mage/assets/34709007/cd53d01c-9844-45d7-81cf-cc2821a35eef)